package.json

```diff
@@ -83,34 +83,37 @@
     "@storybook/preset-typescript": "^3.0.0",
     "@storybook/react": "^5.3.18",
     "@types/classnames": "^2.2.10",
+    "@types/enzyme": "^3.10.5",
     "@types/jest": "^25.1.4",
+    "@types/node": "^14.0.4",
     "@types/react": "^16.9.27",
-    "@typescript-eslint/eslint-plugin": "^2.33.0",
+    "@types/react-dom": "^16.9.8",
+    "@typescript-eslint/eslint-plugin": "^2.26.0",
     "@typescript-eslint/parser": "^2.26.0",
     "auto-changelog": "^2.0.0",
-    "babel-eslint": "^10.1.0",
+    "babel-eslint": "^10.0.3",
```

Comment:
Why the downgrade?

Reply:
Again, probably due to `react-scripts`

Reply:
Nope, this was something else. Reverting!